packages/wrangler/src/vectorize/client.ts

```diff
@@ -97,29 +98,37 @@ export async function updateIndex(
 export async function insertIntoIndex(
 	config: Config,
 	indexName: string,
-	vectors: Array<VectorizeVector>
+	body: FormData
 ): Promise<VectorizeVectorMutation> {
 	const accountId = await requireAuth(config);
+
 	return await fetchResult(
 		`/accounts/${accountId}/vectorize/indexes/${indexName}/insert`,
 		{
 			method: "POST",
-			body: JSON.stringify(vectors),
+			headers: {
+				"Content-Type": "multipart/form-data",
```

Comment:
This should be omitted, so the boundary can be generated and included in the content-type. cc @Skye-31 for spotting this!

Reply:
Interesting: we're sure about this? I check the incoming content type in my test in here and omitting this results in
```
    Expected: "multipart/form-data"
    Received: "text/plain;charset=UTF-8"
```
Which is not the correct type for a multipart request

Reply:
That's an artifact of MSW test setup, I think, which doesn't properly support FormData

Reply:
Does this PR work when manually testing against the API?